.gitignore

```diff
@@ -0,0 +1,5 @@
+/tmp
+/out
+*.cprj
+/RTE
```

Comment:
RTE folder should not be in the ignore list. It still contains configuration files that do not belong to layers.